geps/gep-1709/index.md

```diff
@@ -245,8 +261,8 @@ there are CRDs with different channels, the certification fails specifying that
 it's not possible to run the tests as there are different Gateway API channels
 installed in the cluster. If all the Gateway API `CRD`s have the same version
 and the same channel, the tests can be run and the detected version and channel
-will be set in the `GatewayAPIVersion` and `gatewayAPIChannel` fields of the
-final report. Furthermore, the suite must run all the experimental tests when
+will be set in the `apiVersion` and `apiVersionChannel` fields of
```

Comment:
I think I actually prefer `GatewayAPIVersion` and `GatewayAPIChannel` to avoid any confusion here.